packages/service-integration/src/service_integration/settings.py

```diff
@@ -11,13 +9,22 @@ class Registry(BaseModel):
     password: Optional[SecretStr] = None
 
 
-class UserSettings(BaseSettings):
+class AppSettings(BaseSettings):
 
     DOCKER_REGISTRIES: dict[str, Registry] = {
         "local": Registry(url_or_prefix="registry:5000")
     }
     DEFAULT_REGISTRY: str = "local"
 
+    REGISTRY_NAME: str = Field(
+        default="itisfoundation",
```

Comment:
This will break all the CIs we have.

Previously we had the following:
```
    REGISTRY_NAME: str = Field(
        "", description="name of the registry to use for images, default is Docker Hub"
    )
```
Please leave it to empty string meaning Docker Hub.

Reply:
True. My mistake :-)

But i notice now that this change you added introduces an inconsistency with the previous settings ... I will try to adapt  both.